src/modules/home/components/ContributionInfo.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="flex pt-3 w-76">
+  <div class="flex pt-3 w-[17rem]">
```

Comment:
Oh, this is cool! I think we talked about these [arbitrary values](https://tailwindcss.com/docs/width#arbitrary-values) already.